x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -31,10 +31,12 @@ export type CustomTimelineDataGridBodyProps = EuiDataGridCustomBodyProps & {
   eventIdToNoteIds?: Record<string, string[]> | null;
   eventIdsAddingNotes?: Set<string>;
   onToggleShowNotes: (eventId?: string) => void;
+  rowHeight?: number;
   refetch?: () => void;
 };
 
 const emptyNotes: string[] = [];
+const DEFAULT_UDT_ROW_HEIGHT = 40;
```

Comment:
Yea, I set it to that height originally actually, but I think because of the padding around our row actions it's a bit larger. I'll play around with it some more

Reply:
Done @logeekal . 👍🏾